packages/sdk/react-client/package.json

```diff
@@ -6,48 +6,96 @@
   "bugs": "https://github.com/dxos/dxos/issues",
   "license": "MIT",
   "author": "DXOS.org",
-  "main": "dist/lib/node/index.cjs",
-  "browser": {
-    "./dist/lib/node/index.cjs": "./dist/lib/browser/index.mjs",
-    "./testing.js": "./dist/lib/browser/testing/index.mjs"
+  "exports": {
```

Comment:
did you have to do that by hand? would be cool for esbuild executor to set those

Reply:
yeah, I copy-pasta, but executor doing it might make sense if it turns out to always be the same (which I think it is)